architecture-building-systems/CityEnergyAnalyst
# 1917 create zone shapfile from map

This PR solves #1917 which aims to allow users to create a zone.shp file based on a selection from a map when they create a new scenario. Besides the map tool, a basic landing page interface/flow is added to the dashboard interface, which allow users to access a 'project management' page. The user experience of the landing page could be further improved in #1921.

With the added features, users should now be able to:
- Create a new project in the landing page
- Open an existing project in the landing page
- Create a new scenario in the current project
- Create zone.shp file using the map tool when creating a new scenario
- See the current active project and scenario in the dashboard

To test it:
- Create a new project from the landing page
- Create a new scenario and select the checkbox to create the zone.shp file
- Use the map tool to select a zone

The zone.shp file should be created in the new project directory, together with the age and occupancy files that were generated with default parameters.
